src/main/kotlin/de/bund/digitalservice/useid/widget/QRCodeWidgetController.kt

```diff
@@ -27,14 +24,13 @@ class QRCodeWidgetController(
     ): ModelAndView {
         // no tracking for desktop solution while prototyping
 
-        val widgetViewConfig = mapOf(
-            "baseUrl" to applicationProperties.baseUrl,
+        val modelAttributes = mapOf(
```

Comment:
why the renaming to modelAttributes?

Reply:
To have a consistent naming between the methods indicating thats the same thing. And I don't like the word `config` in there. I find it missleading. Do you prefer the old way?

Reply:
Cleaned up even more now.